tests/test_crash_reports.py

```diff
@@ -19,7 +18,13 @@ class TestCrashReports:
         'B2G']
 
     @pytest.mark.nondestructive
-    @pytest.mark.parametrize(('product'), _expected_products)
+    @pytest.mark.parametrize(('product'), [
+        'Firefox',
+        'Thunderbird',
+        'SeaMonkey',
+        'FennecAndroid',
+        'WebappRuntime',
+        pytest.mark.xfail(reason='bug 1232440')('B2G')])
```

Comment:
The bug reports both SeaMonkey and B2G are affected, but we're only xfailing B2G here.